lib/cli/writer.js

```diff
@@ -23,7 +23,7 @@ const os = require('os');
 const unmount = require('./unmount');
 const imageStream = require('../image-stream');
 const errors = require('../shared/errors');
-const constraints = require('../shared/constraints');
+const constraints = require('../shared/drive-constraints');
```

Comment:
This writer file ties together things like unmounting, and actual writing, so it'd be hard to automatically test without some NUCs and a custom test runner :)